src/pkg/packaging/rpm/dotnet-sharedframework-rpm_config.json

```diff
@@ -32,6 +32,14 @@
   "rpm_dependencies": [{
       "package_name": "%HOSTFXR_RPM_PACKAGE_NAME%",
       "package_version": ""
+    },
+    {
+      "package_name": "libssl1.0.0",
```

Comment:
I am not sure these are the correct package names.  On my RHEL machine, it looks like the packages we want to depend on are:

- openssl-libs
- libcurl

This also seems to match what Fedora is doing (no surprise).